packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -254,15 +281,23 @@ async function handler(req: NextApiRequest & { userId?: number }) {
     });
     updatedBookings = updatedBookings.concat(allUpdatedBookings);
   } else {
+    const updateBookingData = {
+      status: BookingStatus.CANCELLED,
+      cancellationReason: cancellationReason,
+    };
```

Comment:
There's no benefit extracting this into a constant here. It is not being reused anywhere.

https://handbook.cal.com/engineering/best-practices/prefer-inferred-types-over-explicit-ones